db/migrate/20200407141146_create_court_hearings.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCourtHearings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :court_hearings, id: :uuid do |t|
+      t.references :move, type: :uuid, null: false, index: true, foreign_key: true
+
+      t.datetime :start_time
```

Comment:
probably start_time should not be nullable as there is a validation in the class to prevent this